packages/synapse-interface/pages/state-managed-bridge/index.tsx

```diff
@@ -473,6 +474,21 @@ const StateManagedBridge = () => {
       dispatch(removePendingBridgeTransaction(currentTimestamp))
       console.log('Error executing bridge', error)
       toast.dismiss(pendingPopup)
+
+      /** Fetch balances if await transaction receipt times out */
+      if (isTransactionReceiptError(error)) {
+        console.log(
+          'isTransactionReceiptError: ',
+          isTransactionReceiptError(error)
```

Comment:
This always returns true if it's in this `if` block here, correct? Can just log the string without calling the `isTransactionReceiptError(error)` fn again

Reply:
yeah that's right - only meant to include log for testing purposes though, was planning on removing after reviews - doing so now